src/vs/editor/contrib/rename/browser/rename.ts

```diff
@@ -79,6 +79,33 @@ export function rename(model: IReadOnlyModel, position: Position, newName: strin
 	});
 }
 
+function resolveInitialRenameValue(model: IReadOnlyModel, position: Position): TPromise<RenameInitialValue> {
+	const supports = RenameProviderRegistry.ordered(model);
```

Comment:
We should just go for the first rename provider so that we always use the same for resolving the location and for the actual rename.